enzyme/Enzyme/GradientUtils.cpp

```diff
@@ -116,19 +130,30 @@ Value *GradientUtils::unwrapM(Value *const val, IRBuilder<> &BuilderM,
     }
   }
 
-// llvm::errs() << "uwval: " << *val << "\n";
-#define getOp(v)                                                               \
+#define getOpFullest(vtmp, frominst, check)                                    \
```

Comment:
Does it make sense to turn this into an inlined function instead of a macro?

Reply:
This function was apparently a major performance bottleneck, thus why it became a macro. For release builds it doesn't matter, but I presume marking as inline won't actually inline in debug builds?

Reply:
Right I forget that GCC/Clang ignores force inlining in debug mode.